src/App.css

```diff
@@ -1,39 +1,48 @@
-.App {
-  text-align: center;
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+body {
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: center;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.calculator {
+  width: 25rem;
+  margin: 0 auto;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
+.screen {
+  height: 150px;
+  background-color: aquamarine;
+}
+
+.buttons {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  background-color: black;
+  height: 30rem;
+  gap: 0.8rem;
+  padding: 1rem;
```

Comment:
- [ ] Firstly, i must admit you have a very beautiful design 👏 . However, the requirements was to stick to the design to sees replication. you can find the instruction below.
- the idea is to see your skill in exact replication based on a given design. you'll have space to have your personification later 

## Requirement:
![image](https://user-images.githubusercontent.com/50990925/234450219-1fbd3ff6-19db-4c10-ba0c-0b1acf5d3ac7.png)

# Your design:
![image](https://user-images.githubusercontent.com/50990925/234450043-f1189e17-ceae-498a-96ce-22eb45a2a824.png)

## Design given:
![image](https://user-images.githubusercontent.com/50990925/234450092-56048fac-81f3-4585-af9f-dbc4aa065228.png)